riscv/zicfiss.h

```diff
@@ -15,7 +15,7 @@
     shadow_return_addr = MMU.ss_load<uint64_t>(ssp); \
   software_check(reg == shadow_return_addr, SHADOW_STACK_FAULT);
 
-#define store_to_ssp(reg, ssp) \
+#define store_to_ssp(ssp, reg) \
```

Comment:
This change in the 2nd commit suggests that the first commit has a bug? If so this should be rebased back into the first commit, so there's no bug at any point.

Reply:
Thanks for the suggestion. I don't think I need to split it into two patches. I'll squash these two patches.